app/campaigns/campaigns.controller.js

```diff
@@ -0,0 +1,23 @@
+const CampaignsService = require('./campaigns.service')
+
+const CampaignsController = {
+  getCampaigns
+}
+
+async function getCampaigns (request, response) {
+  const status = request.query.status
+  var campaigns = null
+  switch (status) {
```

Comment:
In my opinion, this belongs in the service. The controller should only care about requests and responses, and have no further logic.